packages/node/test/exec.test.ts

```diff
@@ -83,9 +83,12 @@ describe('exec()', () => {
       await converge(() => expect(output).toContain("listening"));
     });
 
-    it('has a pid', () => {
+    it('has a pid', async () => {
       expect(typeof proc.pid).toBe('number');
       expect(proc.pid).not.toBeNaN();
+      // otherwise windows doesn't shut it down and we get
+      // EADDRINUSE: address already in use :::29000
+      await fetch('http://localhost:29000', { method: "POST", body: "exit" });
```

Comment:
If windows doesn't shut it down, then doesn't that mean that it isn't working?

Reply:
I believe it might be a race condition here?